packages/RichTextEditing-Core.package/RichTextEditor.class/instance/isStructureShortcut..st

```diff
@@ -0,0 +1,6 @@
+input events
+isStructureShortcut: aKeyboardEvent
+
+	 ^aKeyboardEvent controlKeyPressed and: [
+		aKeyboardEvent shiftPressed] and: [
```

Comment:
Is Block syntax the best way with and: ? I think `(aKeyboardEvent shiftPressed)` is preferred

Reply:
I think its better to use the block, because without it, you would always evaluate the statement even if statements before already evaluated to false

Reply:
ah ok it was that way